packages/craco/lib/plugin-utils.js

```diff
@@ -0,0 +1,12 @@
+function gitHubIssueUrl(repo, query) {
+    return `https://github.com/${repo}/issues${query ? `?q=is%3Aissue+${query}` : ""}`;
+}
+
+function throwInvalidConfigError({ message, gitHubIssueQuery: query }) {
+    throw new Error(`${message}\n\n Did you update create-react-app or craco recently? \nPlease take a look at some recent issues in craco and create-react-app to see if someone has found a solution: \n\n* ${gitHubIssueUrl("sharegate/craco", query)}\n* ${gitHubIssueUrl("facebook/create-react-app", query)}\n`);
```

Comment:
I use prettier to auto-format my code, but your `.prettierrc.js` has `printWidth: 5000`, which seems a bit crazy. Do you want me to turn off Prettier and break this up into multiple lines, or would you consider changing the `printWidth` to a more reasonable value?

Reply:
I opened a few PRs to solve this problem for future contributors:

* #19 - Set `printWidth` to 80 in `.prettierrc`
* #18 - Turn on word wrapping in the VS Code workspace settings

(My vote is for #19, but up to you!)